bot.py

```diff
@@ -793,6 +844,68 @@ async def broadcast(interaction: discord.Interaction, message: str):
         content="Broadcast message sent to all team channels.", ephemeral=True
     )
 
+
+@app_commands.guild_only()
+@app_commands.default_permissions(administrator=True)
+@bot.tree.command(name="my_team", description="Get information about your current team")
+async def my_team(interaction: discord.Interaction):
+    """
+    Provides information about the user's current team, including team name, members, and team lead.
+
+    Args:
+        interaction (discord.Interaction): The Context of the Interaction.
+    """
+
+    user = interaction.user
+    guild = interaction.guild
+    if not guild:
+        await interaction.response.send_message(
+            content="There was an error retrieving the Discord server information. Please contact an organizer for assistance.",
```

Comment:
This might be a bit of an odd idea, but should we include error codes in each of the three error messages that are the same? If we had an error code like "ERMTNG" ("ERror My_Team No Guild") or something similar, it could help us troubleshoot issues should they ever arise. A person with an issue with ideally show us the error message, which contains an error code, and then we could look to the code to figure out what caused the error and go from there.